ci/activate_creds.sh

```diff
@@ -3,14 +3,18 @@
 # Creates credentials file $1 from two environment variables (see
 # below) which combine to decrypt the keys for a service account.
 # Does gcloud auth using the result.
-if [ ! "${GCLOUD_CREDENTIALS}" ]
-then
-  echo "No GCLOUD_CREDENTIALS env var defined, aborting creds activation."
+
+# Default values for environment variables
+CREDENTIALS_ENV_VAR_NAME=${2:-GCLOUD_CREDENTIALS}
+KEY_ENV_VAR_NAME=${3:-GCLOUD_CREDENTIALS_KEY}
```

Comment:
(copied from #8224)

Allow overriding env var names, but keep the defaults as GCLOUD_CREDENTIALS and GCLOUD_CREDENTIALS_KEY so that other uses of this script don't break. It's possible we can remove the need for defaults entirely, but I'm not confident I won't break an unrelated workflow. I'd need to research this more.